src/main/java/by/custom/utilcalculator/domain/UserProgress.java

```diff
@@ -2,19 +2,18 @@
 
 import by.custom.utilcalculator.domain.constants.steps.*;
 
-public class UserProgress {
+import java.io.Serializable;
+
+public class UserProgress implements Serializable {
     private CountryOrigin countryOrigin = null;
     private OwnersType ownersType = null;
     private CarAge carAge = null;
     private TypeOfEngine typeOfEngine = null;
     private VolumeOfEngine volumeOfEngine = null;
+    private final String chatID;
```

Comment:
Please, for consistency use something one `chatId` or `userId`

Reply:
chatId  is better. I renamed all the variables into the same way